### ISSUE TITLE
Create 'Network' tab

### ISSUE THREAD

**Issue by pgross41** [cerner/f-twelve#38]
Create a 'Network' tab in addition to to the 'Console' tab. Today, the `Console` tab is visible when the tool is hidden. Replace that button with the f-twelve logo and show the tabs only when the tool is shown. 

Can intercept XHR requests with the following (unsure if possible to get static content requests (images, stylesheets, etc.). This is from chrome-extension://hkodokikbjolckghdnljbkbhacbhpnkb/ajaxIntercept.js.  
```js
(function () {
    let XHR = XMLHttpRequest.prototype;
    const open = XHR.open;
    const send = XHR.send;

    XHR.open = function (method, url) {
        this._method = method;
        this._url = url;

        return open.apply(this, arguments);
    };

    XHR.send = function (postData) {
        this.addEventListener('load', function () {
            try {
                const jsonObject = JSON.parse(this.responseText);
                const postDataObject = JSON.parse(postData);
                const details =
                {
                    timeStamp: Date.now(),
                    url: postDataObject['tokenEndpoint']
                };

                if (jsonObject.hasOwnProperty('AccessToken') && jsonObject['AccessToken'] !== null || jsonObject.hasOwnProperty('Error') && jsonObject['Error'] !== null) {
                    window.postMessage({ command: 'messageAjaxTokenPut', fedObject: jsonObject, details: details }, function (response) { return true; });
                }
            }
            catch (e) {

            }
        });

        return send.apply(this, arguments);
    };
})();
```

**Comment by pgross41** [comment]
Also... [Intercepting network requests](https://livebook.manning.com/book/progressive-web-apps/chapter-4/)

**Comment by pgross41** [comment]
Still to do on https://github.com/pgross41/f-twelve/tree/38-network-tab

- Test with CSSA
- Test with PC
- React alias/adapter when used as module?
- Unit tests

**Comment by pgross41** [comment]
Network tab available in 2.2.0
